backend/src/test/java/gov/cdc/usds/simplereport/validators/FileValidatorTest.java

```diff
@@ -49,33 +52,32 @@ void malformedCsv_returnsError() {
   }
 
   @Test
-  void missingHeaders_returnsErrors() {
+  void patientBulkUpload_missingHeaders_returnsErrors() {
     // GIVEN
     InputStream input = loadCsv("patientBulkUpload/missingHeaders.csv");
     // WHEN
     List<FeedbackMessage> errors = patientBulkUploadFileValidator.validate(input);
     // THEN
-    assertThat(errors).hasSize(13);
+    assertThat(errors).hasSize(26);
     List<String> errorMessages =
         errors.stream().map(FeedbackMessage::getMessage).collect(Collectors.toList());
-    List<List<Integer>> indices =
-        errors.stream().map(FeedbackMessage::getIndices).collect(Collectors.toList());
-    assertThat(errorMessages)
-        .contains(
-            "first_name is a required column.",
-            "last_name is a required column.",
-            "race is a required column.",
-            "date_of_birth is a required column.",
-            "biological_sex is a required column.",
-            "ethnicity is a required column.",
-            "street is a required column.",
-            "state is a required column.",
-            "zip_code is a required column.",
-            "phone_number is a required column.",
-            "phone_number_type is a required column.",
-            "employed_in_healthcare is a required column.",
-            "resident_congregate_setting is a required column.");
-    indices.forEach(i -> assertThat(i).isEqualTo(List.of(2)));
+    errors.forEach(
+        error -> {
+          String errorMessage = error.getMessage();
+          if (errorMessage.contains("File is missing data in the")) {
+            assertThat(error.getIndices()).isEqualTo(List.of(2));
+          } else if (errorMessage.contains("The header for column")) {
+            assertThat(error.getIndices()).isEqualTo(List.of(1));
+          }
+        });
+    patientBulkUploadRequiredFields.forEach(
```

Comment:
Iterating over the fields like this is definitely nice and readable, but I'm wondering if it's a little too optimized 😄 In this case, if someone accidentally took something a value away from `requiredFields` (like first_name) the test wouldn't catch it properly, because it's relying on the same constant values to make assertions. It's a little clunkier, but I think the test will be more effective if you maintain a separate list of the required fields within the test suite instead of using the implementation as the source of truth.

Reply:
ahhh!! Good point! I've added the list of required fields back to the test files! Thank you!!